src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -119,6 +119,9 @@ class FormPage extends React.Component {
 
     const pageClasses = classNames('form-panel', route.pageConfig.pageClass);
     const data = this.formData();
+    const formContextWithTrackingPrefix = Object.assign({}, formContext, {
```

Comment:
I think this will end up breaking the forms in surprising ways. In `SchemaForm` we reset some state with the form context changes, since it's not expected to change except when we change pages.

I think I would pass this as its own prop to SchemaForm, then add it to the form context in the `getEmptyState` method of that component.

Reply:
You'll also need to pass this prefix in a couple other places, to make sure it works on the review page. You can look for other uses of SchemaForm, but I believe the two component are ReviewCollapsibleChapter and the review version of ArrayField.

Reply:
@jbalboni @erikphansen, please take another look.

I just pushed my changes per this convo & the 1st (top) resolved one.